src/common/services/recordNormalizing/recordNormalizing.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface IRecordNormalizing {
+  init(record: RecordEntry, block?: string, reference?: RecordReference): void;
```

Comment:
Can this be consistent style-wise with line below it? I.e. 
```suggestion
  init: (record: RecordEntry, block?: string, reference?: RecordReference) => void;
```